mypy.ini

```diff
@@ -9,9 +8,10 @@ disallow_untyped_calls = True
 disallow_incomplete_defs = True
 disallow_untyped_defs = True
 disallow_untyped_decorators = True
-warn_unused_ignores = True
 ignore_missing_imports = False
 
+# mypy-copilot: disabled as mypy only reports it if other checks pass
```

Comment:
I am confused as to why this warrants setting `ward_unused_ignores=False`. For me this change means that mypy doesn't raise an error if all other checks pass except this one. I don't understand why we want that. Do you mind making the comment above a bit clearer?

Reply:
I'd argue we don't necessarily care that much whether the setting is on or off, because unused ignores aren't really errors per se. They are mypy saying "this code type checks just fine, there's no need for a `type: ignore` here." That means that this setting being on or off isn't particularly important, and we can get essentially all the benefit by just flipping it on every few months and cleaning up what it finds.

`False` is just what `typing-copilot` generates at the moment. I can flip it to `True`, we'll just have to keep flipping it back until `typing-copilot` is able to check for it not being necessary.

Reply:
Resolved offline: This comment is autogenerated by the typing copilot utility. I would prefer if the my.ini file was 100% autogenerated so I am ok with keeping this comment as is even though it could be written better.